motionplan/motionPlanner.go

```diff
@@ -85,50 +111,38 @@ func PlanFrameMotion(ctx context.Context,
 	solutionMap, err := motionPlanInternal(
 		ctx,
 		logger,
-		[]*frame.PoseInFrame{destination},
+		destination,
 		f,
 		seedMap,
 		fs,
 		nil,
-		[]map[string]interface{}{planningOpts},
+		constraintSpec,
+		planningOpts,
 	)
 	if err != nil {
 		return nil, err
 	}
 	return FrameStepsFromRobotPath(f.Name(), solutionMap)
 }
 
-// PlanWaypoints plans motions to a list of destinations in order for a given frame. It takes a given frame system, wraps it with a
-// SolvableFS, and solves. It will generate a list of intermediate waypoints as well to pass to the solvable framesystem if possible.
-func PlanWaypoints(ctx context.Context,
```

Comment:
What was PlanWaypoints used for, and why is it being removed?

Reply:
It is not and was never used, and only served as a convenience- but it's actually much less convenient to use than the other entry points. There was never any benefit to using it. Now seemed like a good time to remove it before GA, since preserving it also made implementing constraints more complicated.

Since the motion API only deals with single waypoints, and multiples could be handled by either the Motion or Nav services in the future if we add them, there's not a need for it to exist.